searx/static/themes/oscar/js/searx.min.js

```diff
@@ -1,2 +1,2 @@
-/*! oscar/searx.min.js | 10-08-2020 | https://github.com/asciimoo/searx  */
```

Comment:
Minified files do not belong into git and easily create merge conflicts.

Reply:
See https://github.com/searx/searx/issues/2034

Reply:
@SuperSandro2000 minified files exist in the git repository before this PR. I don't say we shouldn't change this (see @return42 issue), but not in this PR.